.travis.yml

```diff
@@ -26,3 +36,7 @@ script:
     - OMP_NUM_THREADS=1 tests/src/eigenwave3d
     - python tests/eigenwave3d.py default --output
     - python tests/eigenwave3d.py read
+    - python tests/eigenwave3d.py pluto --execute --nthreads=4
+    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
+    - python tests/eigenwave3d.py pluto --output
+    - python tests/eigenwave3d.py read
```

Comment:
Ok, also I'm using pybench.py , which would generate the "results" folder, but if its ok I'll just add it.
by example script do you mean by eigenwave3d.py? not yet but it is fairly easy , should I do it before merge?

Date: Fri, 11 Sep 2015 08:24:25 -0700
From: notifications@github.com
To: opesci-fd@noreply.github.com
CC: zlcghoul@hotmail.com
Subject: Re: [opesci-fd] Pluto (#45)

In .travis.yml:

> @@ -26,3 +36,7 @@ script:
>      - OMP_NUM_THREADS=1 tests/src/eigenwave3d
>      - python tests/eigenwave3d.py default --output
>      - python tests/eigenwave3d.py read
> +    - python tests/eigenwave3d.py pluto --execute --nthreads=4
> +    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
> +    - python tests/eigenwave3d.py pluto --output
> +    - python tests/eigenwave3d.py read

Cool, please add the script to this branch if it is functional already. Does that mean you also have implemented the tile size switch for the example script?


―
Reply to this email directly or view it on GitHub.